### PR TITLE
G0 4315 radio checkbox change

### PR DESCRIPTION
**Pull request by sadik-walkover**
* **What kind of change does this PR introduce?** (Bug fix, feature, docs update, ...)

* **What is the current behavior?** (You can also link to an open issue here)

* **What is the new behavior (if this is a feature change)?**

* **Other information**:

### CODE REVIEW

File: apps/web-giddh/src/app/daybook/export-daybook/export-daybook.component.html [hunk 1]
```diff
@@ -8,36 +8,46 @@ <h3>Export Daybook</h3>
...
-                    <input type="radio" name="emailTypeSelected" [(ngModel)]="emailTypeSelected" [value]="emailTypeMini"
-                        class="radio_theme cp">Mini
-                </label>
-                <label class="radio-inline pd0 mrL2">
-                    <input type="radio" name="emailTypeSelected" [(ngModel)]="emailTypeSelected"
-                        [value]="emailTypeDetail" class="radio_theme cp">Detailed
-                </label>
+
+                <span class="pd0 mrR3">
+                    <input id="Mini" class="radio-custom cp ml0" name="emailTypeSelected" type="radio" [(ngModel)]="emailTypeSelected" [value]="emailTypeMini">
```
Comment (prateek-giddh) [inline apps/web-giddh/src/app/daybook/export-daybook/export-daybook.component.html]: Keep IDs in camelCase

File: apps/web-giddh/src/assets/css/style-2.scss [hunk 2]
```diff
@@ -2223,13 +2235,29 @@ input[type="checkbox"]:after {
...
 input[type="checkbox"]:checked:after {
     background-color: $border-blue;
     border-color: $border-blue;
 }
 
+
+
+
+
+//   input[type="checkbox"]::before {
```
Comment (prateek-giddh) [inline apps/web-giddh/src/assets/css/style-2.scss]: Avoid introduction of commented code and empty lines

File: apps/web-giddh/src/app/ledger/components/exportLedger/exportLedger.component.html [hunk 3]
```diff
@@ -8,36 +8,47 @@ <h3>Export Ledger</h3>
...
-                </label>
+
+                <span class="pd0 mrR3">
+                    <input id="mini" class="radio-custom cp ml0" type="radio" name="emailTypeSelected" [(ngModel)]="emailTypeSelected" [value]="emailTypeMini">
+                    <label for="mini" class="radio-custom-label">Mini</label>
+                </span>
+
+                <span class="pd0 mrR3">
+                    <input id="Detailed" class="radio-custom cp ml0" type="radio" name="emailTypeSelected" [(ngModel)]="emailTypeSelected" [value]="emailTypeDetail">
+                    <label for="Detailed" class="radio-custom-label">Detailed</label>
```
Comment (prateek-giddh) [inline apps/web-giddh/src/app/ledger/components/exportLedger/exportLedger.component.html]: This id is still not camelCase

File: apps/web-giddh/src/app/purchase/purchase-invoice/aside-menu/aside-menu-purchase-invoice-setting.component.html [hunk 4]
```diff
@@ -11,15 +11,20 @@ <h2 *ngIf="selectedService === 'RECONCILE'">Gst Reconcile</h2>
...
+
+
+                <span class="pd0 mrR3">
+                    <input id="vayana" class="radio-custom cp ml0" type="radio" name="serviceType" (change)="changeProvider('VAYANA')"
+                    [checked]="selectedService === 'VAYANA'">
+                    <label for="vayana" class="radio-custom-label">VAYANA</label>
+                </span>
+
+                <span class="pd0 mrR3">
+                    <input id="taxpro" class="radio-custom cp ml0" type="radio" name="serviceType" (change)="changeProvider('TAXPRO')"
```
Comment (prateek-giddh) [inline apps/web-giddh/src/app/purchase/purchase-invoice/aside-menu/aside-menu-purchase-invoice-setting.component.html]: This id is still not camelCase

File: apps/web-giddh/src/app/onboarding/onboarding.component.html [hunk 5]
```diff
@@ -63,16 +63,19 @@ <h1 class="borderbtm">Have a question to ask?</h1>
...
-                        </label>
+
+                        <span class="pd0 mrR3 d-inline-block">
+                            <input id="product" class="radio-custom cp ml0" type="radio" [checked]="CompanySettingsObj?.companyInventorySettings?.manageInventory"
+                            (change)="updateInventorySetting(true)" name="inventoryType">
+                            <label for="product" class="radio-custom-label">Product</label>
+                        </span>
+
+                        <span class="pd0 mrR3 d-inline-block">
+                            <input id="service" class="radio-custom cp ml0" type="radio" [checked]="!CompanySettingsObj?.companyInventorySettings?.manageInventory"
```
Comment (rohit-walkover) [inline apps/web-giddh/src/app/onboarding/onboarding.component.html]: use camel casing CompanySettingsObj

File: apps/web-giddh/src/app/daybook/export-daybook/export-daybook.component.html (hunk 1 above)
Comment (sadik-walkover) [reply apps/web-giddh/src/app/daybook/export-daybook/export-daybook.component.html]: @prateek-giddh  Mini id name changed to mini

File: apps/web-giddh/src/assets/css/style-2.scss (hunk 2 above)
Comment (sadik-walkover) [reply apps/web-giddh/src/assets/css/style-2.scss]: Removed empty lines

File: apps/web-giddh/src/app/ledger/components/exportLedger/exportLedger.component.html (hunk 3 above)
Comment (sadik-walkover) [reply apps/web-giddh/src/app/ledger/components/exportLedger/exportLedger.component.html]: id Detailed name changed to detailed

File: apps/web-giddh/src/app/purchase/purchase-invoice/aside-menu/aside-menu-purchase-invoice-setting.component.html (hunk 4 above)
Comment (sadik-walkover) [reply apps/web-giddh/src/app/purchase/purchase-invoice/aside-menu/aside-menu-purchase-invoice-setting.component.html]: taxpro id name changed to taxPro

**Pull request merged**